config/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	KeyPath          string   `env:"SOFT_SERVE_KEY_PATH"`
 	RepoPath         string   `env:"SOFT_SERVE_REPO_PATH" envDefault:".repos"`
 	InitialAdminKeys []string `env:"SOFT_SERVE_INITIAL_ADMIN_KEY" envSeparator:"\n"`
+	CacheSize        int      `env:"SOFT_SERVE_CACHE_SIZE" envDefault:"0"`
```

Comment:
Just making sure I understand this fully: the `CacheSize` being `0` would mean caching is disabled, as we never call `lru.New` below, right?